evoting/service/service.go

```diff
@@ -740,6 +741,7 @@ func (s *Service) save() {
 	if err := s.Save(storageKey, s.storage); err != nil {
 		log.Error(err)
 	}
+	s.SaveVersion(dbVersion)
```

Comment:
This might overwrite the existing version. Instead of just saving the version, I think we need to:
1. save the data only when current version is 0 or `dbVersion`, write `dbVersion` too if the version is 0
2. otherwise display an error

but I can imagine that this is the first PR out of many, so it's fine to leave it like this for now.

Reply:
actually, we should check for error

Reply:
checking for error is good - I don't see why you don't want to write the version every time we save.

Reply:
It's just a small optimisation, if the version number is the same then there's no need to write it. What I'm imagining is that the upgrade process will store the correct version number after the upgrade. So, if the version number is something different then something went wrong and you don't want to overwrite it.